docs/indexing.md

```diff
@@ -19,18 +19,30 @@ Frequency of updates: once per day
 SCSB ids start with ‘SCSB’
 
 ## Source: Theses
-About 65,000 senior theses pulled from the OIT Dataspace repository http://dataspace.princeton.edu/jspui/handle/88435/dsp019c67wm88m
+About 68,800 senior theses pulled from the OIT Dataspace repository http://dataspace.princeton.edu/jspui/handle/88435/dsp019c67wm88m
 
-Frequency of updates: once per year
+Frequency of updates: Once per year. The Mudd Manuscript Library - Collections Cordinator will contact the Orangelight tech liaison and request the senior theses to be loaded into the catalog.
 
-Thesis record ids start with ‘dsp’
+The https://github.com/pulibrary/orangetheses repository is used to pull the theses from dspace. 
+
+SSH to the bibdata machine that is used for indexing (bibdata-alma-staging) and start a tmux session
```

Comment:
Can we keep just the general info in this section and move lines 28-36 to a new section below under "creating a full index"? This will keep it consistent with the other source info sections.

Reply:
yes we can do that